test/test.html

```diff
@@ -3,6 +3,14 @@
 <head>
     <meta charset="utf-8">
     <title>OpenSeadragon QUnit</title>
+    <script type="text/javascript">
+        window.QUnit = {
+            config: {
+                //one minute per test timeout
+                testTimeout: 60000
```

Comment:
Nice to have this explicit. Is it bigger or smaller than the default?

Reply:
Much bigger. My tests were getting timeout too soon, though one minute might be unnecessary long.

Reply:
I finally remembered. I forced timeouts on the tests (they were not there) because some tests like fullscreen do not work in interactive mode, as the event is blocked as untrusted. Tests never finished so I added timeout. Now, when you debug a test, timeout that was like one second or so made debugging impossible, since the test failed before you got to the point where the target issue was. 60 seconds was for development... I will try to figure out how to do this more dynamically...

Reply:
Great, thank you!